homeassistant/components/keyboard_remote/__init__.py

```diff
@@ -18,11 +18,13 @@
 DEVICE_ID_GROUP = "Device description"
 DEVICE_NAME = "device_name"
 DOMAIN = "keyboard_remote"
+VALUE = "value"
 
 ICON = "mdi:remote"
 
 KEY_CODE = "key_code"
 KEY_VALUE = {"key_up": 0, "key_down": 1, "key_hold": 2}
+KEY_VALUE_NAME = {0: "key_up", 1: "key_down", 2: "key_hold"}
```

Comment:
To not make mistakes, invert the map programmatically instead, so something like:

```suggestion
KEY_VALUE_NAME = {value: key for key, value in KEY_VALUE .items()}
```